platforms/software/ivy/src/main/java/org/gradle/api/publish/ivy/IvyPublication.java

```diff
@@ -183,6 +186,7 @@ public interface IvyPublication extends Publication {
      * Returns the complete set of configurations for this publication.
      * @return the configurations
      */
+    @ToBeKeptEagerProperty(because = "Container property")
     IvyConfigurationContainer getConfigurations();
```

Comment:
❗ Should not need annotation as it's a `DomainObjectContainer`.

Reply:
Done